nav2_safety_nodes/src/nav2_safety_node.cpp

```diff
@@ -0,0 +1,250 @@
+#include <chrono>
+#include <memory>
+#include <vector>
+#include <string>
+#include <utility>
+
+#include "nav2_safety_nodes/nav2_safety_node.hpp"
+
+using namespace std::chrono_literals;
+
+namespace nav2_safety_nodes
+{
+
+SafetyZone::SafetyZone()
+: nav2_util::LifecycleNode("SafetyZone", "", false)
+{
+  logger_ = get_logger();
+  RCLCPP_INFO(logger_, "Creating Safety Polygon");
+
+  // Vector of string for multiple LaserScan topics
+  const std::vector<std::string> scan{};
```

Comment:
This variable should be called `scan_topics` and it should contain a single entry `scan` so by default it uses the scan topic if none other are given